src/constants.h

```diff
@@ -48,6 +48,9 @@ CONSTEXPR(uint32_t, CAPTIVEPORTAL_REQUEST_TIMEOUT_MSEC, 10000, 4000, 0)
 // How fast the animated icon should move
 CONSTEXPR(uint32_t, STATUSICON_ANIMATION_MSEC, 200, 200, 0)
 
+// How often glean pings are sent
+CONSTEXPR(uint32_t, GLEAN_TIMEOUT_MSEC, 3600000, 1000, 0)
```

Comment:
May I ask why this scheduling was chosen? I am wondering since the metric collected right now is the `appStarted` metric, scheduling can be simplified a bit and only instead send the ping once everytime the app is started. 

Note that the ping is only sent when there are metrics to send (because it is declared with `sendIfEmpty: false`), since `appStarted` is only set on start, subsequent pings will not be sent either way :)

Reply:
@bakulf I'm curious about this, too :-)